src/WorldWind.js

```diff
@@ -567,6 +567,18 @@ define([ // PLEASE KEEP ALL THIS IN ALPHABETICAL ORDER BY MODULE NAME (not direc
             FAILED: "failed",
 
             /**
+             * Indicates a linear filter.
+             * @constant
+             */
+            FILTER_LINEAR: "filter_linear",
```

Comment:
The prefix FILTER was used in order to differentiate this constant from the other LINEAR constant in this class. In the first commit, I had actually created an enum class for holding these two, but moved to the standard approach afterwards. This might be a solution to consider for organising these constants in the future.